Module: Resizeable Message Pane

Argavyon opened the pull request:
As asked in https://www.nexusclash.com/viewtopic.php?f=14&t=2068

Argavyon committed:
Module: Resizeable Message Pane
As asked in https://www.nexusclash.com/viewtopic.php?f=14&t=2068

Argavyon committed:
Axed a stray console.log

AnneTrue commented:
I believe this duplicates the global message history module's undocumented behaviour:
- On page loads with more than 10 lines, it creates a longer message box (250 px).
- It changes the message box to allow resizing vertically: https://github.com/AnneTrue/nexus-clash-interface-tweaks/blob/v1.7.0/nexus-tweaks.user.js#L705

Argavyon commented:
> I believe this duplicates the global message history module's undocumented behaviour:
> 
> * On page loads with more than 10 lines, it creates a longer message box (250 px).
> * It changes the message box to allow resizing vertically: https://github.com/AnneTrue/nexus-clash-interface-tweaks/blob/v1.7.0/nexus-tweaks.user.js#L705

I'll try to triple-check, but it's not easy to come upon a 10+ lines messagebox, sadly.

Argavyon committed:
Bugfix (Message Pane Resize)
Prevented resizing of the full message log on the character page.

Argavyon committed:
Minor Feature: Message Pane Resize
Added checkbox for automatic pane size
Added default value for message-pane-height setting
